UnitsNet.Tests/CustomCode/ThermalResistanceTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated (once) by \generate-code.bat, but will not be
+//     regenerated when it already exists. The purpose of creating this file is to make 
+//     it easier to remember to implement all the unit conversion test cases.
+// 
+//     Whenever a new unit is added to this unit class and \generate-code.bat is run,
+//     the base test class will get a new abstract property and cause a compile error
+//     in this derived class, reminding the developer to implement the test case
+//     for the new unit.
+//
+//     See https://github.com/anjdreas/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\UnitClasses\MyUnit.extra.cs files to add code to generated unit classes.
+//     Add Extensions\MyUnitExtensions.cs to decorate unit classes with new behavior.
+//     Add UnitDefinitions\MyUnit.json and run GeneratUnits.bat to generate new units or unit classes.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Copyright (c) 2007 Andreas Gullberg Larsen (anjdreas@gmail.com).
+// https://github.com/anjdreas/UnitsNet
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class ThermalResistanceTests : ThermalResistanceTestsBase
+    {
+        protected override double HourSquareFeetDegreesFahrenheitPerBtuInOneSquareMeterKelvinPerKilowatt => 0.0056782d;
+        protected override double SquareCentimeterHourDegreesCelsiusPerKilocalorieInOneSquareMeterKelvinPerKilowatt => 11.63088897992627d;
+        protected override double SquareCentimeterKelvinsPerWattInOneSquareMeterKelvinPerKilowatt => 10.00035223670306d;
+        protected override double SquareMeterDegreesCelsiusPerWattInOneSquareMeterKelvinPerKilowatt => 9.999119516790815e-4;
```

Comment:
Yes, these inputs I got it from my domain expert. For your reference:
1)	Attached is the page I scanned from Heat Exchanger Design Hand Book.
2)	Second below is the link to one of the web site. 
  http://www.asknumbers.com/HeatTransferCoefficientConversion.aspx
 
Initially  I got it based on base unit hr*ft2°F/Btu. 

Base Unit = hr*ft2°F/Btu
 
m2 oC /W = 5.6787 (hrft2°F/Btu)                          hrft2°F/Btu = 0.1760966 (m2 oC/W) 
cm2K/W = 0.0005678 (hrft2°F/Btu)                 hrft2°F/Btu = 1761.1 (cm2K/W)
cm2hr oC /kcal = 0.0004882 (hrft2°F/Btu)           hrft2°F/Btu = 2048.16 (cm2hroC/kcal)
m2K/kW = 0.0056782 (hrft2°F/Btu)                hrft2°F/Btu = 176.11 (m2K/kW)

and when you suggested me to use SquareMeterKelvinPerKilowatt (m2K/kW) as base unit, I converted it based on above and confirmed with our domain expert.

Base Unit = SquareMeterKelvinPerKilowatt (m2K/kW)

hrft2°F/Btu = 176.1121482159839 m2K/kW

m2 oC /W = 1000.088056074108 m2K/kW

cm2K/W = 0.0999964777570357 m2K/kW

cm2hr oC /kcal = 0.0859779507590433 m2K/kW

and below values are based on above values.
namespace UnitsNet.Tests.CustomCode
{
    public class ThermalResistanceTests : ThermalResistanceTestsBase
    {
        protected override double HourSquareFeetDegreesFahrenheitPerBtuInOneSquareMeterKelvinPerKilowatt => 0.0056782d;
        protected override double SquareCentimeterHourDegreesCelsiusPerKilocalorieInOneSquareMeterKelvinPerKilowatt => 11.63088897992627d;
        protected override double SquareCentimeterKelvinsPerWattInOneSquareMeterKelvinPerKilowatt => 10.00035223670306d;
        protected override double SquareMeterDegreesCelsiusPerWattInOneSquareMeterKelvinPerKilowatt => 9.999119516790815e-4;
       protected override double SquareMeterKelvinsPerKilowattInOneSquareMeterKelvinPerKilowatt => 1;

    }
}

In first example, 
hrft2°F/Btu = 176.1121482159839 m2K/kW
so,  m2K/kW = 1/176.1121482159839 = 0.0056782d hrft2°F/Btu

in the second example,
cm2hr oC /kcal = 0.0859779507590433 m2K/kW
so, m2K/kW = 1/ 0.0859779507590433 = 11.63088897992627 cm2hr oC /kcal

[Scanned from a Xerox Multifunction Printer.pdf](https://github.com/anjdreas/UnitsNet/files/914127/Scanned.from.a.Xerox.Multifunction.Printer.pdf)